internal/kuberesource/mutators.go

```diff
@@ -528,7 +522,7 @@ func MapPodSpecWithMeta(
 	f func(
 		meta *applymetav1.ObjectMetaApplyConfiguration,
 		spec *applycorev1.PodSpecApplyConfiguration,
-	) (*applymetav1.ObjectMetaApplyConfiguration, *applycorev1.PodSpecApplyConfiguration),
+	),
```

Comment:
We briefly talked about the alternative: actually use the return values to update the meta/spec. That would allow setting an existing pointer to nil, which the version without return value does not allow. Do you still think we should be removing the return value?